readme.md

```diff
@@ -37,24 +37,20 @@ Type: `Object`
 ##### timeout
 
 Type: `number`<br>
-Default: `2000`
+Default: `5000`
 
-Milliseconds to wait for a server to respond. This option is only supported in Node.js.
+Milliseconds to wait for a server to respond.
 
-##### hostnames
+##### version
```

Comment:
I usually prefer making decisions for the users so they don't have, but sure, we can leave it. Just add a note in the option that it's for advanced users and you most likely don't need it.